libraries/AP_HAL_ChibiOS/hwdef/NucleoH743-IKS02/hwdef-bl.dat

```diff
@@ -0,0 +1,49 @@
+# hw definition file for processing by chibios_hwdef.py
```

Comment:
This should `include` the hwdef.dat from the board this is derived from, `undef APJ_BOARD_ID` and set that alone.